addons/web_editor/static/src/js/widgets.js

```diff
@@ -830,11 +830,8 @@ function createVideoNode(url, options) {
         });
     } else {
         // this is not a known video link. Now what, Cat? Now what?
-        $video = $('<iframe>', {
-            width: '640',
-            height: '360',
-            src: url
-        });
+        $video = $('<iframe>');
```

Comment:
Compared to v11 here the check on valid URLs is already done in the IF blocks above. Here we should simply not render the frame w/ the given URL.
As this could break expected behaviors from customers, I added a log entry.

Any feedback?

Reply:
Cat? :joy_cat: (about comment above)

More seriously, seems legit. v11 only returns result of super method for unknown url without adding the iframe and here we definitively don't want to put the untrusted src in the iframe.